app/section/sections/browser-additional-information/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "id": "browser-additional-information",
+  "title": "Browser based - additional information",
+  "mainAdvice": "Answer the following questions about your browser based solution",
+  "additionalAdvice": [
+    "This information is diplayed on the Marketing page for buyers to view"
+  ],
+  "questions": {
+    "additional-information": {
+      "mainAdvice": "Do you have any additional browser based information (optional)",
+      "additionalAdvice": "Add any additional information or requirement your Solution needs to function",
+      "footerAdvice": "You have 500 characters remaining",
```

Comment:
Sorry this isn't right. Can you change to the following `"You can enter up to 500 characters"` - I think the prototype was out of date